### PR TITLE
Handle exceptions inside commands

### PR DESCRIPTION
**Pull request by Chemaclass**
## 📚 Description

Don't throw up the logic exceptions from the commands, but handle them inside the commands themselves.

## 🔖 Changes

- Remove the `@throws ...` from the commands. They don't intentionally throw an exception up.
- Catch and handle the exceptions inside the `command->run()` logic.

### CODE REVIEW

**Review by JesusValera — APPROVED** [review]
👍🏼 💯

**Comment by jenshaase** [comment]
Can we also write the Stack Traces? Currently only the messages of the exceptions are printed. May we should add `writeException` method to the IO class, since this is not the first time it was forgotten?

**Comment by Chemaclass** [comment]
I will take a look at it later. Thanks for the idea 💡

File: src/php/Command/CommandFacade.php [hunk 1]
```diff
@@ -51,21 +36,12 @@ public function executeRunCommand(string $fileOrPath): void
...
-     *
-     * @throws CompilerException
-     * @throws CompiledCodeIsMalformedException
-     * @throws FileException
-     * @throws CannotFindAnyTestsException
      */
     public function executeTestCommand(array $paths): void
     {
-        $result = $this->commandFactory
+        $this->commandFactory
```
Comment (jenshaase) [inline src/php/Command/CommandFacade.php:42]: As far as I can see this isn't sending no output codes (SUCCESS_CODE, FAILED_CODE) anymore? This is required for the CI Pipeline to indicate that the tests failed.

**Review by jenshaase — CHANGES_REQUESTED** [review]

File: src/php/Command/CommandFacade.php (hunk 1 above)
Comment (Chemaclass) [reply src/php/Command/CommandFacade.php:42]: Ups, I didn't know this. Ok, I will rollback this as how it was before 🪂

**Review by jenshaase — APPROVED** [review]

**Comment by jenshaase** [comment]
Thanks. It is merged now!

**Pull request merged**